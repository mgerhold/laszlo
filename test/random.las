let seed = 42;

let modulus = 65536;
let a = 25213;
let c = 11;
let current = seed;

let next = 0;
let results = [];
for _ in 0..600 {
    current = (a * current + c) mod modulus;
    results = results + [current mod 6 + 1];
}

for i in 1..=6 {
    let count = 0;
    for j in 0..600 {
        if results[j] == i {
            count = count + 1;
        }
    }
    println(i + ": " + count);
}
