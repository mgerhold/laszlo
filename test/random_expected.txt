1: 95
2: 100
3: 107
4: 96
5: 98
6: 104
